snapshot/testsuite/testsuite.go

```diff
@@ -18,8 +18,29 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type SnapshotterSuiteOpt func(*snapshotterSuiteOpts) error
+
+type snapshotterSuiteOpts struct {
+	TestViewReadonly bool
+}
+
+// WithTestViewReadonly ensures a KindView snapshot to be mounted as a read-only filesystem.
```

Comment:
Because it is valid to have writable KindView snapshot

https://github.com/containerd/containerd/blob/7d3a1e77376c612a4d0b9a06d1efa97d3b957e54/snapshot/snapshotter.go#L226-L230

```
	// This method operates identically to Prepare, except that Mounts returned
	// may have the readonly flag set. Any modifications to the underlying
	// filesystem will be ignored. Implementations may perform this in a more
	// efficient manner that differs from what would be attempted with
	// `Prepare`.
```